### DISCUSSION TITLE
What is the argument for client.actor() ?

### DISCUSSION THREAD

**Discussion by timonson** [apify/apify-client-js#528] [Q&A]
I can't find a good explanation inside the documentation what the string for `await.client.actor()` is. It is a string for a resource. But what resource is this. What does this string mean? Is it unique or secret? Where do I find it? Thanks! Would you mind elaborating please. Thank you!

```js
import { ApifyClient } from "apify-client";

// Initialize the ApifyClient with API token
const client = new ApifyClient({
  token: "apify_api_dku1ovxcbvcvbvcbnxvgnfgjhf6h3ETo5C",
});

// Prepare Actor input
const input = {
  "usernames": [
    "horst_gmbh",
  ],
};

(async () => {
  // Run the Actor and wait for it to finish
  const run = await client.actor("dSCLg0C3YEZ8ghzYX").call(input);

  // Fetch and print Actor results from the run's dataset (if any)
  console.log("Results from dataset");
  const { items } = await client.dataset(run.defaultDatasetId).listItems();
  items.forEach((item) => {
    console.dir(item);
  });
})();
```

**Comment by B4nan** [comment]
Its ID of the actor or its unique name, see https://docs.apify.com/academy/getting-started/apify-client

**Comment by timonson** [comment]
So it means this string is just for me to identify the actor? It can be just any UUID, right?

**Reply by B4nan** [reply]
No, its an ID of an existing actor on the apify platform.
